source/crud/tests/v2/findOneAndDelete-hint-clientError.yml

```diff
@@ -0,0 +1,38 @@
+runOn:
+  - { maxServerVersion: "4.0.99" }
```

Comment:
Updated. There was no change to the serverError tests, but I replaced 4.1.0 here with 4.0.99 and removed the TMI comment. Since `maxServerVersion` is inclusive, specifying a high patch value seemed the most sensible way to signal <4.2.